src/python_testing/TC_CADMIN_1_22_24.py

```diff
@@ -104,13 +106,14 @@ async def test_TC_CADMIN_1_22(self):
         except ChipStackError as e:
             # Since we provided 901 seconds as the timeout duration,
             # we should not be able to open comm window as duration is too long.
+            # we are expected receive Failed to open commissioning window: IM Error 0x00000585: General error: 0x85 (INVALID_COMMAND)
             asserts.assert_equal(e.err,  0x00000585,
                                  "Expected to error as we provided failure value for opening commissioning window")
 
         self.step(6)
         window_status2 = await self.get_window_status()
         if window_status2 != Clusters.AdministratorCommissioning.Enums.CommissioningWindowStatusEnum.kWindowNotOpen:
```

Comment:
This if statement can still be removed above

Reply:
Thank you, this has been completed!